packages/roosterjs-content-model-core/lib/coreApi/setDOMSelection/setDOMSelection.ts

```diff
@@ -51,12 +51,20 @@ export const setDOMSelection: SetDOMSelection = (core, selection, skipSelectionC
                     type: 'image',
                     image,
                 };
+                let imageSelectionColor =
+                    core.selection.imageSelectionBorderColor || DEFAULT_SELECTION_BORDER_COLOR;
+                if (imageSelectionColor !== DEFAULT_SELECTION_BORDER_COLOR && isDarkMode === true) {
```

Comment:
Instead of calculating the color every time, can we just calculate it once, and save the result?

For example, when initialize selection plugin, we do
state.imageSelectionBorderColor = option.imageSelectionBorderColor ?? default_color;
state.imageSelectionBorderColorDark = option.imageSelectionBorderColor ? this.editor.getDarkColor(option.imageSelectionBorderColor) : default_color;

Reply:
Updated!